builder/core/data.py

```diff
@@ -338,6 +338,48 @@ class PKG_TOOLS(Enum):
             'osx_architectures': 'arm64;x86_64'
         },
     },
+    'tvos': {
+        'cmake_args': [
+            '-GXcode',
+            '-DCMAKE_SYSTEM_NAME=tvOS',
+            '-DCMAKE_OSX_ARCHITECTURES="{osx_architectures}"',
+            '-DCMAKE_OSX_DEPLOYMENT_TARGET={osx_deployment_target}'
+        ],
+        'run_tests': False,
+        'architectures': {
+            'armv8': {
+                'variables': {
+                    'osx_architectures': 'arm64'
+                }
+            }
+        },
+        'variables': {
+            'exe': '',
+            'osx_deployment_target': '13.0',
+            'osx_architectures': 'arm64;x86_64'
+        },
+    },
+    'watchos': {
+        'cmake_args': [
+            '-GXcode',
+            '-DCMAKE_SYSTEM_NAME=watchOS',
+            '-DCMAKE_OSX_ARCHITECTURES="{osx_architectures}"',
+            '-DCMAKE_OSX_DEPLOYMENT_TARGET={osx_deployment_target}'
+        ],
+        'run_tests': False,
+        'architectures': {
+            'armv8': {
+                'variables': {
+                    'osx_architectures': 'arm64'
+                }
+            }
+        },
+        'variables': {
+            'exe': '',
+            'osx_deployment_target': '13.0',
```

Comment:
same here. think this should be 5.0

Reply:
tvOS tracks iOS, this one is still right :)